README.md

```diff
@@ -7,6 +7,118 @@ height="42" />
 
 Runtime for Newfold WP modules and plugins
 
+## Module Responsibilities
+
+* Container is loaded from the brand plugin in Runtime class and it returns a `NewfoldRuntime` object. 
```

Comment:
We need to clarify the differences between the `NewfoldRuntime` object that gets output to JavaScript via PHP and the `NewfoldRuntime` object that is made available via the NPM package. This has caused great confusion and several bugs and should be documented. We should consider renaming one to avoid further issues.

Reply:
Updated content.